origin.spec

```diff
@@ -101,7 +101,13 @@ Requires:       docker-io >= %{docker_version}
 Requires:       tuned-profiles-%{name}-node = %{version}-%{release}
 Requires:       util-linux
 Requires:       socat
+# storage plugins
+Requires:       e2fsprogs
+Requires:       xfsprogs
 Requires:       nfs-utils
+Requires:       ceph-common
+Requires:       glusterfs-fuse
+Requires:       iscsi-initiator-utils
```

Comment:
Well, unless you disable those storage plugins, right? Anyway, I'm not going to fight this one. I can see both sides, so if you are on the 'just list it' side, I think that's fine.